terraform/testcases/xrayreceiver_mock/parameters.tfvars

```diff
@@ -6,4 +6,4 @@ soaking_data_mode = "trace"
 # data model type. possible values: otlp, xray, etc
 soaking_data_type = "xray"
 
-sample_app_image = "public.ecr.aws/aws-otel-test/aws-otel-goxray-sample-app:v1.2.0"
+sample_app_image = "public.ecr.aws/aws-otel-test/aws-otel-goxray-sample-app:v1.3.0"
```

Comment:
Is there a reason we aren't just using the latest?

Reply:
Merging for now to get ci passing we can discuss later